### ISSUE TITLE
✅ 🔗 🚀 Video widget - YouTube video does not appear if I use a share link from the "Live" list or from the "Shorts" list on any Youtube channel (Duplicate of #20330) 

### ISSUE THREAD

**Issue by mllemata** [elementor/elementor#28720]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

 YouTube video does not appear if I use a share link from the "Live" list or from the "Shorts" list on any Youtube channel

### Steps to reproduce

1. add a Video widget to the page
2. select source: YouTube
3. add link from Live Youtube list - for example https://www.youtube.com/live/n8gc0PU1AUQ?si=X7asziwgnZktyQmI or from Shorts YouTube list for example https://youtube.com/shorts/vj4JTcKCjkQ?si=0w2IGXmzmOVOy_hM
4. publish
5. instead of the player, only a white space is displayed.

### Expected behavior

If you add a link from the "Videos" list, the link and Video widget works correctly. For example https://youtu.be/n_a_Ru1FbtA?si=bskxhxM8_FvM9Ngj

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

```txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.5.22
	PHP Version: 8.3.10
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: https://firmtech.sk
	Home URL: https://firmtech.sk
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: sk_SK
	Timezone: Europe/Bratislava
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.1.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: sk-SK
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.4
		Author: WordPress Contributors

	Disable Comments
		Version: 2.4.6
		Author: WPDeveloper

	Elementor
		Version: 3.24.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.24.1
		Author: Elementor.com

	SVG Support
		Version: 2.5.8
		Author: Benbodhi

	Under Construction
		Version: 4.01
		Author: WebFactory Ltd

	Wordfence Security
		Version: 7.11.7
		Author: Wordfence


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrácia ==
	


== Experimenty so softvérom Elementor ==
	Vylepšené načítavanie CSS: Predvolené ako neaktívne
	Ikony inline písma: Predvolené ako aktívne
	Dodatočné prispôsobiteľné hraničné body: Predvolené ako aktívne
	Kontajner Flexbox: Predvolené ako aktívne
	Grid Container: Predvolené ako aktívne
	Upgrade Swiper Library: Predvolené ako aktívne
	Nested Elements Performance: Predvolené ako aktívne
	Optimized Control Loading: Predvolené ako aktívne
	Optimized Markup: Predvolené ako neaktívne
	Conditionally load Swiper CSS files: Predvolené ako neaktívne
	Plugin Onboarding: Predvolené ako aktívne
	Default to New Theme Builder: Predvolené ako aktívne
	Header & Footer: Neaktívne
	Elementor Home Screen: Predvolené ako aktívne
	Build with AI: Neaktívne
	Vstupné stránky: Predvolené ako neaktívne
	Launchpad Checklist: Predvolené ako neaktívne
	Nested Elements: Aktívne
	Editor Top Bar: Predvolené ako aktívne
	Pages Panel: Predvolené ako neaktívne
	Element Caching: Predvolené ako aktívne
	Link In Bio: Predvolené ako aktívne
	Floating Buttons: Predvolené ako aktívne
	Elementor Editor Events: Predvolené ako neaktívne
	Atomic Widgets: Predvolené ako neaktívne
	Display Conditions: Neaktívne
	Form Submissions: Neaktívne
	Menu: Predvolené ako neaktívne


== Záznam == 

Log: showing 10 of 102024-09-14 15:37:22 [info] elementor::elementor_updater Started 
2024-09-14 15:37:22 [info] Elementor/Upgrades - _on_each_version Start  
2024-09-14 15:37:22 [info] Elementor/Upgrades - _on_each_version Finished 
2024-09-14 15:37:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.4',
  'to' => '3.24.2',
)]
2024-09-14 15:37:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.4',
  'to' => '3.24.2',
)]
2024-09-14 15:38:17 [info] elementor-pro::elementor_pro_updater Started 
2024-09-14 15:38:17 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-09-14 15:38:17 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-09-14 15:38:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.3',
  'to' => '3.24.1',
)]
2024-09-14 15:38:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.23.3',
  'to' => '3.24.1',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Nekompatibilné

== Elementor Pro - Compatibility Tag ==

```
</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @mllemata!

Thank you for reaching out! 🙏 

### ⛔ 🚀 This is more of a Feature Request, not a bug. Elementor's Video widget does not natively support YouTube playlist links or YouTube Shorts links.

Moreover, 🔗 🚀 There is already a feature request submitted by someone in the following threads:

- #20330

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hi there!

We have great news! 🙌 

### 📢 We're happy to announce that YouTube Shorts support for the Video widget was released with Elementor Core v3.27.0! 🥳 

You can now share YouTube shorts in the **Video widget**, and even change the aspect ratio. Just select the **`Source`** as **`YouTube`**, and paste your YouTube shorts link. Then in the **`Style`** tab you can change the Aspect Ration to 9:16.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
